services/cosi-driver-nutanix/0.0.4/defaults/cm.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloudnative-pg-0.23.0-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |-
+    cosiController:
+      enabled: false # This should be deployed during k8s cluster creation by konvoy.
+    objectstorageProvisionerSidecar:
+      image:
+        registry: gcr.io
+        # We can keep this in sync with the sidecar that is deploy in CephCOSIDriver to avoid duplicate images in airgapped bundle.
+        repository: k8s-staging-sig-storage/objectstorage-sidecar
+        tag: v20240513-v0.1.0-35-gefb3255
```

Comment:
Won't this version have CVEs? Can we use latest?

Reply:
looks like the scanner didn't pickup any CVEs (this image has been on main for a ~week). Bumped it to latest anyway in ceph chart and here.